src/main/java/InfrastructureManager/AdvantEdge/AdvantEdgeClient.java

```diff
@@ -0,0 +1,144 @@
+package InfrastructureManager.AdvantEdge;
+
+import InfrastructureManager.MasterOutput;
+import InfrastructureManager.Utils.FileParser;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse;
+import java.nio.file.Paths;
+import java.time.Duration;
+
+public class AdvantEdgeClient implements MasterOutput {
+    private final String requestPath = "http://localhost:10500"; //The controller API is exposed on port 80 & 443 of the node where AdvantEDGE is deployed.
```

Comment:
I believe port 10500 is for testing only, if this is the case, we need to set this variable in the configuration as well